web/views.py

```diff
@@ -7,17 +7,20 @@
 
 import json
 import os
+import random
 
 # from .models import Question
 
 def index(request):
     with open("web/thesauruses/meta_info.json", 'r') as meta_file:
         meta_data = meta_file.read()
     meta_data_langs = json.loads(meta_data)["languages"].keys
+    three_random_langs = random.sample(meta_data_langs(), k=3)
 
     content = {
         'title': 'Welcome',
-        'languages': meta_data_langs
+        'languages': meta_data_langs,
+        'exampleLanguages': three_random_langs
```

Comment:
I get what you're doing here. I'm fine with it, but `exampleLanguages` is a not descriptive name for this variable. Perhaps just keep `random_languages` or `featured_languages` or `highlighted_languages`?